heroku_deploy/entrypoint.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/sh -l
+
+send_chat_message()
+{
+  chat_path="/chat.sh"
+
+  type=$1
+  message=$2
+
+  sh -c "$chat_path $type \"$message\""
+}
+
+abort()
+{
+    echo "...error!"
+    echo ""
+    echo ""
+
+    message="*MANAGEMENT API*: Heroku deploy. Please go to project *${GITHUB_REPOSITORY}* -> Actions to see the errors."
+    type="failed"
+    send_chat_message "$type \"$message\""
+
+    exit 1
+}
+
+trap 'abort' 0
+
+set -e
+
+main(){
+  branch=$(jq --raw-output .pull_request.head.ref ${GITHUB_EVENT_PATH})
+
+  echo "-------------------------------------"
+  echo "-------------------------------------"
+  echo "||                                 ||"
+  echo "||        Deploy to heroku         ||"
+  echo "||                                 ||"
+  echo "-------------------------------------"
+  echo "-------------------------------------"
+
+  echo ""
+  echo ""
+  echo "Deploy ${branch}..."
+
+  if [ -z "${DEPLOY_ENVIRONMENT}" ] || [ "${DEPLOY_ENVIRONMENT}" != "production" ]; then
```

Comment:
I think that this is better to put on the workflow directly with an `if` condition, because by this way, the task is not executed and its more clear to understand on the workflow.

Reply:
it's (will be) already on the workflow (https://github.com/loyalguru/loyal-guru-api/pull/3183/files), but i prefer to keep this additional check for now

Reply:
Okey, perfect, I agree.